src/components/Homes/Carousel.vue

```diff
@@ -1,5 +1,13 @@
 <template>
+
   <div>
+    <div class="icon-bar">
+ <a href="https://www.facebook.com/iiest.shibpur.official" class="facebook"><i class="fab fa-facebook-f"></i></a> 
+  <a href="https://twitter.com/iiest_s" class="twitter"><i class="fab fa-twitter"></i></a> 
+  <a href="https://iiests.irins.org/" class="vidwan"><i class="fas fa-lightbulb"></i></a> 
+  <a href="https://www.linkedin.com/school/iiests/" class="linkedin"><i class="fab fa-linkedin-in"></i></a>
+  <a href="shivamhere247@gmail.com" class="gmail"><i class="  fas fa-envelope"></i></a> 
```

Comment:
make it `href="mailto:shivamhere247@gmail.com"`

Reply:
@praneetha3108  Nice idea! Should we remove the top-right hamburger now? If yes, add it in this patch itself

Reply:
> @praneetha3108 Nice idea! Should we remove the top-right hamburger now? If yes, add it in this patch itself

yeah I will add those hamburger kinks in this

Reply:
@praneetha3108 Should we merge this patch and you can remove those other links in a separate patch, perhaps?